package.json

```diff
@@ -11,7 +11,7 @@
     "start": "NODE_ENV=development  DEBUG=* webpack --mode development --config-name server && node dist/server.js",
     "build": "NODE_ENV=production NODE_OPTIONS=--max_old_space_size=8192  webpack --mode production",
     "test": "jest",
-    "test:watch": "jest --watch",
+    "test:watch": "jest --watch --maxworkers=4",
```

Comment:
There's a typo in the `maxWorkers` argument. The `w` in workers should be captialized ([reference](https://jestjs.io/docs/cli#--maxworkersnumstring))